deploy/bundle/manifests/cephblockpoolradosnamespace.crd.yaml

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
not expected to backport this crd, let's see if we can restrict to only the changes needed for the log collector